api/v1alpha1/llamastackdistribution_types.go

```diff
@@ -39,7 +39,7 @@ const (
 	// DefaultLabelValue is the default value for labels
 	DefaultLabelValue = "llama-stack"
 	// DefaultMountPath is the default mount path for storage
-	DefaultMountPath = "/.llama"
+	DefaultMountPath = "/opt/app-root/src/.llama/distributions/rh/"
```

Comment:
I believe this is where our patching strategy should start? Otherwise every sync will override this?

Reply:
Yes true